src/jit/lower.cpp

```diff
@@ -4775,9 +4775,10 @@ void Lowering::CheckCall(GenTreeCall* call)
 //                      after lowering.
 //
 // Arguments:
+//   compiler - the compiler context.
 //   node - the node to check.
 //
-void Lowering::CheckNode(GenTree* node)
+void Lowering::CheckNode(Compiler* compiler, GenTree* node)
```

Comment:
Another alternative, a little cleaner, but slightly less efficient I suppose, would be to make this an instance method of `Lowering`, which has a `Compiler` member.